roles/teams/tasks/main.yml

```diff
@@ -57,7 +56,6 @@
   changed_when: "gsh_output.stderr != ''"
   args:
     chdir: /opt/www/grouper-shell
-  when: env == 'vm'
```

Comment:
We're now running grouper provisioning on every teams deploy. Is that necessary? I think the "when" was added precisely to avoid messing with grouper provisioning in already installed envs.